go/cmd/vtctldclient/command/backups.go

```diff
@@ -70,7 +70,7 @@ If no replica-type tablet can be found, the backup can be taken on the primary i
 	}
 	// RestoreFromBackup makes a RestoreFromBackup gRPC call to a vtctld.
 	RestoreFromBackup = &cobra.Command{
-		Use:                   "RestoreFromBackup [--backup-timestamp|-t <YYYY-mm-DD.HHMMSS>] [--restore-to-pos <pos>] [--dry-run] <tablet_alias>",
+		Use:                   "RestoreFromBackup [--backup-timestamp|-t <YYYY-mm-DD.HHMMSS>] [--restore-to-pos <pos>] [--ignored-backup-engines=enginename,] [--dry-run] <tablet_alias>",
```

Comment:
I don't like this. Can we not default to using the physical backup unless a specific engine name is provided? IOW, a flag like `backup-engine` to this command instead of `ignored-....`

Reply:
👍 I agree that an allow-list is preferable to a deny-list. Also, use a `StringSliceVar`.